### DISCUSSION TITLE
collection/time-complexity

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#130] [Announcements]
# collection/time-complexity

Java程序员进阶之路，小白的零基础Java教程，Java 时间复杂度

https://tobebetterjavaer.com/collection/time-complexity.html

**Comment by qinple** [comment]
时间复杂度果然是我一生之敌, ...之一?

**Reply by itwanger** [reply]
GitHub 上优化了一版新的，理解起来会更容易一些。

**Comment by Admin027813** [comment]
二哥，还得是你，我在b站看视频一直没明白，看这篇文章明白了。
